corehq/apps/userreports/reports/filters/choice_providers.py

```diff
@@ -186,6 +186,13 @@ def get_values_for_query(self, query_context):
     def get_choices_for_known_values(self, values, user):
         return []
 
+    def _make_choice_from_value(self, value):
+        if value is None:
+            return Choice(NONE_CHOICE, '[Missing]')
+        elif not value:
```

Comment:
Does this cause issues with other falsy values like 0 or false, which could be in UCRs